### PR TITLE
Improve Getting Started by basic information

### PR DESCRIPTION
**Pull request by Krysztophe**
Try to give a very basic view of the different components of the OPM suite.

It is NOT obvious for people discovering OPM/check_pgactivity.

### CODE REVIEW

File: general/getting_started.rst [hunk 1]
```diff
@@ -1,2 +1,30 @@
...
+It is technically separated from OPM.
+
+
+
+Requirements & Installation
+-----------------------------
+
+We suppose that you have in place and configured to your need:
+* a working Nagios (or similar ) installation
+* check_pgactivity (<https://github.com/OPMDG/check_pgactivity>)
```
Comment (rjuju) [inline general/getting_started.rst]: on a local build of the doc,  the `<` and `>` remains.  I'm not specialist of rst syntax, but the syntax doesn't look good, and the list is wrong.

File: general/getting_started.rst [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+or other agents monitoring your PostgreSQL instances
+* an empty PostgreSQL database (9.3 or later) on the instance of your choice
+to store the data.
+
+See the :doc:`Installation <../opm-core/Installation>` page to install:
+* the :ref:`opm_core <opm_core>` and :ref:`wh_nagios <wh_nagios>` extensions
```
Comment (rjuju) [inline general/getting_started.rst]: same here, this syntax doesn't produce the expected list.

File: general/getting_started.rst [hunk 3]
```diff
@@ -1,2 +1,29 @@
...
+
+
+Requirements & Installation
+-----------------------------
+
+We suppose that you have in place and configured to your need:
+  * a working Nagios (or similar ) installation
+  * check_pgactivity (https://github.com/OPMDG/check_pgactivity) or other agents monitoring your PostgreSQL instances
+  * an empty PostgreSQL database (9.3 or later) on the instance of your choice
+to store the data.
```
[the remark this answers is not in the document]
Comment (Krysztophe) [reply general/getting_started.rst]: Sorry (again), fixed

**Comment by rjuju** [comment]
@Krysztophe sorry for the multiple change request, I didn't notice all the issues first (and second) time.  By the way, did you encounter any issue for generating the doc?  I realize that the readme is actually wrong, I'll fix it soon.

**Comment by Krysztophe** [comment]
No real issue, just too much time spent to understand the .rst syntax and logic, that would have better spent to check what I had written...

**Comment by rjuju** [comment]
I manually merged this PR, there was still other issues (windows style EOL and too long lines).

Thanks

**Pull request closed without merging**